rm/src/lib.rs

```diff
@@ -84,16 +86,31 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         .join("namespaces")
         .join(options.urn.encode_id());
 
-    if !options.prompt
+    if options.prompt
```

Comment:
Wouldn't this lead to the project being removed when the user does not confirm (presses `n`)?

Reply:
Sorry, I fear the `fixup!` commit is also not doing the right thing. So right now, if the user confirms the removal, nothing happens. Is there a specific reason you changed the logic here?

Reply:
First of all, thanks for spotting this!

Yes the fixup was wrong as well.  The logic should remain the same, trying to improve the code by isolating the user confirmation to a few lines.